src/GraphControl/Control/Grapher.cpp

```diff
@@ -1040,6 +1043,34 @@ void Grapher::OnGraphBackgroundPropertyChanged(Windows::UI::Color /*oldValue*/,
     }
 }
 
+
+void Grapher::OnGridLinesColorPropertyChanged(Windows::UI::Color /*oldValue*/, Windows::UI::Color newValue)
+{
+    if (m_renderMain != nullptr && m_graph != nullptr)
+    {
+        auto gridLinesColor = Graphing::Color(newValue.R, newValue.G, newValue.B, newValue.A);
+        m_graph->GetOptions().SetGridColor(gridLinesColor);
```

Comment:
This one has the namespaces because of the 2 Color types Windows::UI::Color and Graphing::Color. The other methods for background and axis color also have the namespaces.

Reply:
I tested this out and the gridlines color does not get cleared when a new equation is added.